packages/nx/src/command-line/nx-commands.ts

```diff
@@ -303,7 +303,12 @@ export const commandsObject = yargs
   })
   .command({
     command: 'init',
-    describe: 'Adds nx.json file and installs nx if not installed already',
+    describe: `Adds Nx to any type of workspace. It performs the following changes:
+
+- Installs nx
+- Creates an nx.json configuration file
+- Sets up distributed caching (optional)
+- If migrating an Angular CLI, CRA or NestJS workspace, it installs the relevant Nx plugins for better developer experience`,
```

Comment:
@juristr @isaacplmann I'd welcome any suggestion to make this better.

I thought about including links to the individual pages migrating different types of workspaces, but I wasn't sure if we wanted to do that given this is also shown in the terminal help. I didn't see any other command with links, so I wasn't sure if that was something we were avoiding or if it was that other commands don't need any links.

Reply:
There are a few other terminal commands that use links, but I would only use one link and shorten the description.

Reply:
@isaacplmann I don't think links are rendered in the description. I made the change and added a link but it's not rendered as a link. It's just plain text. Which other commands use links in their description? I couldn't find one.

Reply:
Maybe I'm thinking of create-nx-workspace: packages/create-nx-workspace/src/utils/preset/point-to-tutorial-and-course.ts

Reply:
Yeah, that only appears in the terminal. @bcabanes is there a way to get a link rendered in the command docs page description?

Reply:
After talking to Ben about it, we'll keep the link as plain text until the doc generation is reworked to support such scenarios.